http/src/main/resources/reference.conf

```diff
@@ -306,11 +306,11 @@ azure {
       {
         name = "cromwell-reader"
         kind = "ClusterIP"
+        path = "/cromwell"
       }
     ]
     # Not enabled to make sure people cannot start the (not yet complete) app:
     enabled = false
```

Comment:
Should this be enabled in this PR? If not, do we have a ticket to enable it?

Reply:
Not sure -- you could also add an EV to `leo.conf` (`enabled = ${?WORKFLOWS_APP_ENABLED}`) so we could turn it on in lower environments (dev, bees, etc).

Reply:
OTOH you could just enable it, which seems to be happening from Cromwell Runner in https://github.com/DataBiosphere/leonardo/pull/3621 :)

Reply:
I like the idea of setting this to true for Dev and BEEs via terra-helmfile charts. This way users can't unexpectedly create these new apps in Prod but we can easily test it in Dev or BEEs.

Reply:
Agreed, thanks both!

Reply:
https://github.com/broadinstitute/terra-helmfile/pull/4527